Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                              </tool_sheds>
+                            ```
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Connect your Galaxy to a Tool Shed
+                            - Set dependencies resolution
+                              - in `config/galaxy.ini`:
+                            ```
+                            conda_prefix = <tool_dependency_dir>/_conda/
+                            conda_auto_install = True
+                            conda_auto_init = False
+                            tool_dependency_dir = ../tool_dependencies
+                            ```
+                              - in config/dependency_resolvers_conf.xml (be careful order matters):
+                            ```
+                            <dependency_resolvers>
+                                  <conda />
+                                  <conda versionless="true" />
+                                  <tool_shed_packages />
+                                  <galaxy_packages />
+                                  <galaxy_packages versionless="true" />
+                            </dependency_resolvers>
+                            ```
+                            - Restart Galaxy
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - Go to the admin interface and click on "Search Tool Shed"
+                            <img src="../images/admin_searchtoolshed_ss.png" class="no-border" >
+                            - Select a Tool Shed
+
+                            <img src="../images/goToToolshed_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - Search your tool
+                            <img src="../images/searchTool_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - <span style="font-size:70%">Types of repository</span>
+                              - <span style="font-size:55%">Tool (orange): contains tools (tool wrappers)</span>
+                              - <span style="font-size:55%">Tool dependencies (red): contains only the file tool\_dependencies.xml.
+                              !! tool\_dependencies.xml file is no longer supported because of the use of conda!
```

Comment:
tool_dependencies are still supported but not recommended
The old tool_dependencies will be still understood by Galaxy.